Javascriptdemo2/demo3.js

```diff
@@ -0,0 +1,50 @@
+document.getElementById("submitBtn").addEventListener("click", () => {
+    const input = document.getElementById("TakingInput").value;
+    const number = parseInt(input);
+
+    if (number <= 1) {
+        console.log("It is neither prime or composite number");
+        return;
+    }
+
+    Prime1(number);
+});
+
+function prime(number) {
+    if (number <= 1) {
+        return false;
+    }
+    for (let i = 2; i < number; i++) {  
+        if (number % i === 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+function Prime1(input) {
```

Comment:
Here also, Use arrow functions and use proper naming convention for functions. Don't use like Prime1 it doesn't have a meaning.